SQL/tgstation_schema.sql

```diff
@@ -3,6 +3,7 @@
 /*!50503 SET NAMES utf8mb4 */;
 /*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
 /*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+SET GLOBAL event_scheduler=ON
```

Comment:
This should likely be in the changelog

Reply:
I got rid of the set, set commands dont persist. Dunno how to describe required mariadb configs

Reply:
That being said, it does display a warning if you try to add an even without the event scheduler being turned on